tests/test_enterprise/api/test_views.py

```diff
@@ -420,6 +421,68 @@ def test_post_enterprise_customer_user_logged_out(self):
         response = self.client.post(settings.TEST_SERVER + ENTERPRISE_LEARNER_LIST_ENDPOINT, data=data)
         assert response.status_code == 401
 
+    @ddt.data(
+        (True, True, True, False, 204),
+        (True, True, False, True, 204),
+        (True, False, False, True, 204),
+        (True, False, False, False, 201),
+        (True, True, False, False, 201),
+        (False, False, False, False, 403)
```

Comment:
Nit: you can write these as
```python
{'is_staff': True, 'user_exists': True, 'ecu_exists': True, 'pending_ecu_exists': False, 'status_code': 204},
...
```
and @ddt.unpack will still unpack everything into the function parameters you have.  Definitely more verbose, but way easier to understand at a glance.